roles/ceph-osd/templates/ceph-osd.service.j2

```diff
@@ -5,8 +5,10 @@ After=docker.service
 [Service]
 EnvironmentFile=-/etc/environment
 ExecStartPre=-/usr/bin/docker stop {{ ansible_hostname }}-osd-dev%i
-ExecStartPre=-/usr/bin/docker rm -f  {{ ansible_hostname }}-osd-dev%i
-ExecStart=/usr/bin/docker run --rm --net=host --pid=host\
+ExecStartPre=-/usr/bin/docker rm -f {{ ansible_hostname }}-osd-dev%i
+ExecStart=/usr/bin/docker run --rm --net=host \
+   --cap-add SYS_ADMIN \
```

Comment:
Yes it seems it only allows mounting/unmounting/remounting filesystems within the container, which is exactly why we enabled that cap.
Also see `CAP_SYS_ADMIN` here: http://man7.org/linux/man-pages/man7/capabilities.7.html